src/Simplex/Chat.hs

```diff
@@ -2428,6 +2387,50 @@ processChatCommand = \case
         cReqHashes = bimap hash hash cReqSchemas
         hash = ConnReqUriHash . C.sha256Hash . strEncode
 
+prepareGroupMsg :: forall m. ChatMonad m => User -> GroupInfo -> MsgContent -> Maybe ChatItemId -> Maybe FileInvitation -> Maybe CITimed -> Bool -> m (MsgContainer, Maybe (CIQuote 'CTGroup))
```

Comment:
prepareGroupMsg, quoteContent are moved practically without changes, only parameters are slightly changed to accommodate preparing history message